### PR TITLE
Moved button out of top component

### PR DESCRIPTION
**Pull request by mimse**
Fixing https://github.com/Energinet-DataHub/greenforce-frontend/issues/1172

### CODE REVIEW

File: libs/ui-watt/src/lib/components/drawer/watt-drawer-topbar.component.ts [hunk 1]
```diff
@@ -14,21 +14,11 @@
...
-  template: `
-    <ng-content></ng-content>
-    <watt-button
-      class="close-btn"
-      variant="icon"
-      icon="close"
-      (click)="closed.emit()"
-    ></watt-button>
-  `,
+  template: ` <ng-content></ng-content> `,
```
Comment (dzhavat) [inline libs/ui-watt/src/lib/components/drawer/watt-drawer-topbar.component.ts:21]: nitpick:

```suggestion
  template: `<ng-content></ng-content>`,
```

File: libs/ui-watt/src/lib/components/drawer/watt-drawer.component.html [hunk 2]
```diff
@@ -26,7 +26,13 @@
   >
     <div class="watt-drawer__grid-container">
       <header>
-        <ng-content select="watt-drawer-topbar"></ng-content>
+        <ng-content select="watt-drawer-topbar"> </ng-content>
```
Comment (dzhavat) [inline libs/ui-watt/src/lib/components/drawer/watt-drawer.component.html:29]: nitpick:

```suggestion
        <ng-content select="watt-drawer-topbar"></ng-content>
```

File: libs/ui-watt/src/lib/components/drawer/watt-drawer.component.html [hunk 3]
```diff
@@ -26,7 +26,13 @@
...
+        <watt-button
+          class="close-btn"
+          variant="icon"
+          icon="close"
+          (click)="closed.emit()"
```
Comment (dzhavat) [inline libs/ui-watt/src/lib/components/drawer/watt-drawer.component.html:34]: issue: looks like using the `closed` property directly doesn't close the drawer (have tested it in DH app). Better to call `close` method instead.

```suggestion
          (click)="close()"
```

**Review by frontend-specialisten — APPROVED** [review]
Praise: nice changes! 
I've tested this on wholesale and the close button now works as expected :)

**Pull request merged**